service/siteService.js

```diff
@@ -78,3 +89,35 @@ transformHours = function(time) {
 	}
 }
 
+transformPublicTransit = function(publicTransitOptions) {
+	return publicTransitOptions ? getPublicTransit(publicTransitOptions) : undefined
+}
+
+getPublicTransit = function(publicTransitOptions) {
+	let options = []
+	if(publicTransitOptions) {
+		publicTransitOptions.forEach(function(transitOption) {
+			const properties = transitOption.split("-")
```

Comment:
This is kind of a janky way of doing it, but it was the workaround I found so that we don't have to use up extra database rows

Definitely open to suggestions

Reply:
Is the issue with extra DB rows dealing with queries and reads?

As an aside, it may be worth documenting what the expected cell formats look like.

Reply:
No, everything from the api comes back as nested JSON so that's not too tough (actually would be easier of it were actual queries). The issue is that the free version of airtable only allows 1200 records...so I don't want to depend on having rows for other things when we might need them for sites down the line. Esp. cause for this we would need a lot of rows... 

Very good point about noting the expected format. I will add that!